compiler/daml-lf-reader/src/DA/Daml/LF/Reader.hs

```diff
@@ -29,6 +30,16 @@ lineToKeyValue line = case splitOn ":" line of
     [l, r] -> (trim l , trim r)
     _ -> error $ "Expected two fields in line " <> line
 
+appendToFirstEntry :: [String] -> String -> [String]
+appendToFirstEntry (h : t) nextLine = (h ++ nextLine) : t
+appendToFirstEntry _ _ = error "Reading Manifest file from dar failed."
+
+multiLineContent :: [String] -> [String] -> [String]
```

Comment:
This whole `multilineContent` thing looks way too complicated to me. How about something like
```haskell
multilineContent :: [String] -> [String]
multilineContent [] = []
multilineContent (x:xs)
  | all isSpace x = multilineContent xs
  | otherwise = (x ++ concatMap trim ys) : multilineContent zs
    where
      (ys, zs) = span (isPrefixOf " ") xs
```

Reply:
I went with foldl.

Reply:
I'm not a big fan of `appendToFirstEntry` because it is a partial function when there's no need for a partial function. :(

Reply:
I see, git it. now not using a partial function